tensorflow/python/data/kernel_tests/from_tensors_test.py

```diff
@@ -134,6 +141,7 @@ def testFromTensorsMixedRagged(self):
     self.assertDatasetProduces(dataset, expected_output=[components])
 
   # pylint: disable=g-long-lambda,unnecessary-lambda
+  @combinations.generate(test_base.default_test_combinations())
   def testNestedStructure(self):
```

Comment:
This test does not logically belong to `from_tensors_test`. Move it to `dataset_test.py` and call it `testLegacyStructureAPI`.